Javascript & Ajax/DeleteStudentController.cls

```diff
@@ -0,0 +1,13 @@
+public with sharing class DeleteStudentController {

```

Comment:
no need of controller
refer this :-  http://help.taskfeedapp.com/knowledgebase/articles/1108435-custom-mass-delete-board-button-classic-only

Reply:
In  this assignment we have to call apex remote action to delete selected students. So controller is used.